File: Seccion 11 - Funciones en C++/Seccion_11_Python/01_ejemplo_de_funcion.py

def encontrar_max(x, y):
    if x > y:
        num_max = x
    else:
        num_max = y
    
    return num_max

n1 = int(input('Digite el primer número: '))
n2 = int(input('Digite el segundo número: '))

mayor = encontrar_max(n1, n2)

print('El mayor de los dos números es: {}'.format(mayor))

File: Seccion 11 - Funciones en C++/Seccion_11_Dart/01_ejemplo_de_funcion.dart

import 'dart:io';

int encontrarMax(int x, int y) {
  int numMax;

  if (x > y) {
    numMax = x;
  } else {
    numMax = y;
  }

  return numMax;
}

void main() {

  int n1, n2;
  int mayor;

  stdout.writeln('Digite el primer número: ');
  n1 = int.parse(stdin.readLineSync());
  stdout.writeln('Digite el segundo número: ');
  n2 = int.parse(stdin.readLineSync());

  mayor = encontrarMax(n1, n2);

  print('\El mayor de los dos números es: $mayor\n');

}